pynsee/utils/_request_insee.py

```diff
@@ -91,18 +94,8 @@ def _request_insee(
 
             code = results.status_code
             
-            if "status_code" not in dir(results):            
+            if "status_code" not in dir(results):
                 success = False
-            elif code == 429:
```

Comment:
why was this removed?

Reply:
This was changed in the latest release and after this branch's creation.